src/components/ui/nav/index.tsx

```
@@ -37,73 +37,87 @@ import {
+    }
+
+    return (
+      <LoginDialog>
+        <button className="flex items-center gap-2 text-sm p-2">
+          <svg
+            width="23"
+            height="23"
+            fill="none"
+            xmlns="http://www.w3.org/2000/svg"
+            className="fill-foreground scale-[0.75]"
+          >
```

Biome (2.1.2)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)